app/views/actual_db_schema/migrations/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Migrations</title>
+    <%= stylesheet_link_tag 'actual_db_schema/styles', media: 'all' %>
+  </head>
+  <body>
+    <div>
+      <h2>Migrations</h2>
+      <p>
+        <span style="background-color: #ffe6e6; padding: 0 5px;">Red rows</span> represent phantom migrations.
+      </p>
+      <div class="link">
+        <%= link_to 'Phantom Migrations', phantom_migrations_path %>
+      </div>
+      <% if migrations.present? %>
+        <table>
+          <thead>
+            <tr>
+              <th>Status</th>
+              <th>Migration ID</th>
+              <th>Name</th>
+              <th>Branch</th>
+              <th>Database</th>
+              <th>Actions</th>
+            </tr>
+          </thead>
+          <tbody>
+            <% migrations.each do |migration| %>
+              <tr class="migration-row <%= migration[:phantom] ? 'phantom' : 'normal' %>">
+                <td><%= migration[:status] %></td>
+                <td><%= migration[:version] %></td>
+                <td><%= migration[:name] %></td>
+                <td><%= migration[:branch] %></td>
+                <td><%= migration[:database] %></td>
+                <td>
+                  <div class='button-container'>
+                    <%= link_to '👁 Show',
+                                migration_path(id: migration[:version], database: migration[:database]),
+                                class: 'button' %>
+                    <%= button_to '⎌ Rollback',
+                                  rollback_migration_path(id: migration[:version], database: migration[:database]),
+                                  method: :post,
+                                  class: 'button',
+                                  disabled: migration[:status] == "down" %>
+                    <%= button_to '⬆ Migrate',
+                                  migrate_migration_path(id: migration[:version], database: migration[:database]),
+                                  method: :post,
+                                  class: 'button',
+                                  disabled: migration[:status] == "up" || migration[:phantom] %>
```

Comment:
@VladislavSokov please apply a few changes here:

1. Show only the relevant button: either Rollback or Migrate depending on the migration status
2. The "Name" column takes up almost the whole space if the migration name is too long. E.g. `AddCutOffTimeAndAmTreatmentWindowAndTimezoneAndMinimumBufferDaysAndAmTreatmentOptionAndAlertMessageToCountryConfigurations`

<img width="1497" alt="image" src="https://github.com/user-attachments/assets/56138ec9-8195-493a-8eaf-4151360d6de9">

Let's truncate the name with the possibility to show the whole name on hover.

Reply:
3. Change the sorting to DESC. The most recent migrations should be on top: for both phantom and usual ones.

Reply:
4. The "Phantom migraions" tab is supposed to be too useful. But it's kinda hidden in the UI. It's barely clear that it's even clickable. Can we improve that so that it's clear that a user needs to click this button in order to show only phantom migrations?

<img width="1497" alt="image" src="https://github.com/user-attachments/assets/a52bfb69-667a-4f0c-9ea6-8a4497d3aeec">